package.json

```diff
@@ -15,7 +15,7 @@
     "@types/node": "^6.0.46",
     "@types/q": "^0.0.32",
     "@types/selenium-webdriver": "~2.53.39",
-    "blocking-proxy": "0.0.5",
+    "blocking-proxy": "^0.0.5",
```

Comment:
I'm a bit worried about allowing `^` since this is pre version 1 and versioning is nebulous -can we leave it precisely pinned?